docs/guidance/rayservice.md

```diff
@@ -153,12 +157,12 @@ serveDeploymentStatuses:
 
 After it finishes deployment, let's send a request again. In the curl pod from earlier, run:
 ```shell
-[ root@curl:/ ]$ curl -X POST -H 'Content-Type: application/json' rayservice-sample-serve-svc.default.svc.cluster.local:8000 -d '["MANGO", 2]'
+[ root@curl:/ ]$ curl -X POST -H 'Content-Type: application/json' rayservice-sample-serve-svc.default.svc.cluster.local:8000/fruit/ -d '["MANGO", 2]'
```

Comment:
We need to tell users why do we need to add `/fruit/` here.

Reply:
I've added explanations!